### DISCUSSION TITLE
switch (true) pattern

### DISCUSSION THREAD

**Discussion by sbraccia** [parcel-bundler/parcel#6991] [Q&A] [answered]
I am having problems with how parcel 2.0.0-rc.0 compiles _switch (true) {...}_ pattern. 

For instance the code below

![image](https://user-images.githubusercontent.com/54271863/135053669-1edb82a3-3540-4439-aa72-f7b3845bc939.png)

gets compiled to just the default branch:

![image](https://user-images.githubusercontent.com/54271863/135053724-aa7a51a2-8d9d-4486-9a42-2866c436eb00.png)

I was wondering if this is a known issue or if I am doing something wrong (I am running parcel with zero configuration). 

Thank you in advance for your feedback

**Comment by mischnic** [comment, marked as answer]
Please try using a nightly version (`yarn add parcel@nightly`). It seems to be fixed already

**Reply by sbraccia** [reply]
Thank you mischnic for the super quick response, the nightly (843) seems to be unhappy about something that I cannot figure out.. why is it looking for a serve file?

![image](https://user-images.githubusercontent.com/54271863/135063601-8728e96e-9cd9-4f0e-9fe7-e6bf0e355dc7.png)

**Reply by sbraccia** [reply]
Sorry, nevermind, it was my mistake after all. Yes the nightly has fixed the switch (true) issue.

Thank you very much
